### ISSUE TITLE
Parallel Translation searches

### ISSUE THREAD

**Issue by averagehat** [DCGenomics/seqr#21]
Does it make sense to make a Solr request for each sequence frame concurrently? 
I am not sure if Embedded Solr might make better use of all cores automatically.

**Comment by lianyi** [comment]
Yes, we can configure how many searcher thread can be used per core. I
remember it's 2 in current setting.

-lianyi

**Comment by averagehat** [comment]
Good, that should be a commandline setting if possible; I'll look into it as we are about to do some checkpointing of seqr against blast/diamond once I can get the databases to be equivalent.
